helm-charts/falcon-kac/values.yaml

```diff
@@ -147,6 +150,13 @@ webhook:
   # Comma sparated list of namespaces in which we need to disable validation e.g test1,test2
   disableNamespaces:
 
+# Specifies if Falcon KAC should use hostNetwork mode. This is required in some scenarios such as when a 
+# custom CNI is in use where control plane nodes cannot establish network communication with pods.
+hostNetwork: false
+
+# Define Falcon KAC POD DNS Policy, defaults to "ClusterFirstWithHostNet" when hostNetwork = true
+dnsPolicy:
```

Comment:
@TomRyan-321 can you set the dnsPolicy default to `ClusterFirstWithHostNet` here instead?

Reply:
@TomRyan-321 right. So, the request is to set the default here instead of in the webhook.yaml

Reply:
Understood updating